<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowT</class>
 <widget class="QDialog" name="mainwindowT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="enName"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="w_Position"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="w_box">
     <item>
      <property name="text">
       <string>North</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>South</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="j_Position"/>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="j_box">
     <item>
      <property name="text">
       <string>East</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>West</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="population"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="checkCity">
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="addCity">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>              城市信息录入</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
